src/.vuepress/theme/styles/layout/_nav.scss

```diff
@@ -83,20 +83,23 @@
     padding: 0;
     list-style: none;
 
-
     .sub-menu:not(.no-parent) {
       padding-left: 16px;
       transform: scaleY(0);
       transform-origin: 50% 0;
-      transition:transform 0.2s linear, height 0.2s linear, opacity 0.2s linear;
-      opacity:0;
+      transition: transform 0.2s linear, height 0.2s linear, opacity 0.2s linear;
+      opacity: 0;
       height: 0px;
     }
 
     .displaySubmenu {
       opacity: 1 !important;
       transform: scaleY(1) !important;
     }
+
+    & > .md-nav__item-container {
```

Comment:
It looks like that in mobile view (See First Steps): 

![image](https://user-images.githubusercontent.com/44427849/61617139-660ab780-ac6a-11e9-81b1-71ae207dbfba.png)